app/models/ambassador_task_assignment.rb

```diff
@@ -0,0 +1,29 @@
+class AmbassadorTaskAssignment < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :ambassador_task
+
+  validates :user, presence: true
+  validates :ambassador_task, presence: true
+
+  validates :ambassador_task, uniqueness: { scope: :user }
+  validate :associated_user_is_an_ambassador
+
+  delegate :description, :description_html, to: :ambassador_task
+
+  default_scope { order(created_at: :asc) }
```

Comment:
I used to use default scopes regularly. It turns out [they're a pain](https://andycroll.com/ruby/dont-use-default-scope/) and difficult to remove. I would recommend avoiding using them.